content/en/docs/next/op-guide/monitoring.md

```diff
@@ -129,6 +129,23 @@ Sample dashboard:
 
 ![](../etcd-sample-grafana.png)
 
+## Distributed tracing
+
+In v3.5 etcd has added support for distributed tracing using [OpenTelemetry](https://github.com/open-telemetry).
+
+> Note: This feature is still experimental and can change at any time.
+
+To enable this experimental feature, pass the `--experimental-enable-distributed-tracing=true` to the etcd server. Setup and configure the distributed tracing by starting etcd server with the following flags:
```

Comment:
Since the following are all optional, it may be clearer to state it here and remove the "(Optional)" from the flags below
```suggestion
To enable this experimental feature, pass the `--experimental-enable-distributed-tracing=true` to the etcd server. Setup and configure the distributed tracing by starting etcd server with the following optional flags:
```